components/Alert/Alert.tsx

```diff
@@ -1,24 +1,23 @@
-import { func, node, oneOf, string } from 'prop-types';
 import classNames from 'classnames';
 import { ALERT, ALERT_CLOSE_BUTTON } from 'common/constants/testIDs';
 import ScreenReaderOnly from 'components/ScreenReaderOnly/ScreenReaderOnly';
 import styles from './Alert.module.css';
 
-Alert.propTypes = {
-  children: node.isRequired,
-  className: string,
-  'data-testid': string,
-  onClose: func,
-  type: oneOf(['error', 'success', 'warning']).isRequired,
+export type AlertPropsType = {
+  type: 'error' | 'success' | 'warning';
+  children: React.ReactNode;
+  className?: string;
+  'data-testid'?: string;
+  onClose?: () => void;
 };
 
-Alert.defaultProps = {
-  className: undefined,
-  'data-testid': ALERT,
-  onClose: undefined,
-};
-
-function Alert({ children, className, 'data-testid': testID, onClose, type }) {
+function Alert({
+  children,
+  className = undefined,
+  'data-testid': testID = ALERT,
+  onClose = undefined,
```

Comment:
```suggestion
  className,
  'data-testid': testID = ALERT,
  onClose,
```

Whenever you see `undefined` as a default prop, you don't need to supply it. That was an annoyance with `prop-types` that TS does not need to worry about. The default value of an undefined field is inherently `undefined`. It's also good to not do this because defautl values DO show up in the bundle, so we're needlessly increasing bundle size.